packages/components/src/color-control/README.md

```diff
@@ -0,0 +1,61 @@
+# ColorControl
+
+ColorControl is used to generate a color picker user interface.
+
+
+## Usage
+
+Render a user interface to change fixed background setting.
+```jsx
+import { ColorControl } from '@wordpress/components';
+import { withState } from '@wordpress/compose';
+
+const MyColorControl = withState( {
+	backgroundColor: '#fff',
+} )( ( { backgroundColor, setState } ) => (
+	<ColorControl
+			label={ 'Background Color' }
+			color={ backgroundColor }
+			onChange={ val => setState( { backgroundColor: val.hex } ) }
+			disableAlpha
+		/>
```

Comment:
There’s some funky indentation happening in here. It’ also be nice to see `val` as `value` just to be more explicit.

Reply:
Update.